quipucords/cli/utils.py

```diff
@@ -98,3 +98,20 @@ def pretty_print(json_data):
     """
     return json.dumps(json_data, sort_keys=True, indent=4,
                       separators=(',', ': '))
+
+
+# Read in a file and make it a list
+def read_in_file(filename):
+    """Read values from file into a list object. Expecting newline delimited.
+
+    :param filename: the filename to read
+    :returns: the list of values found in the file
+    """
+    result = None
+    input_path = os.path.expanduser(os.path.expandvars(filename))
+    try:
+        with open(input_path, 'r') as in_file:
+            result = in_file.read().splitlines()
```

Comment:
Small note: `in_file.readlines()` does this.

Reply:
Calling that function gives back each line with the newline character still present.
`['1.2.3.4\n', '1.2.3.[1:10]\n']`

The above gives me back `['1.2.3.4', '1.2.3.[1:10]']` which I'm looking for and is less processing than then stripping of the newline character.

Reply:
@chambridge I didn't know that! Thanks for explaining.